Bruin Bite/View Controllers/MenuVC.swift

```diff
@@ -46,13 +57,27 @@ class MenuVC: UIViewController, UICollectionViewDelegate, UICollectionViewDataSo
 
         
     override func viewDidLoad() {
+        
+        loading_1 = UIImage(named: "anim1")
+        loading_2 = UIImage(named: "anim2")
+        loading_3 = UIImage(named: "anim3")
+        loading_4 = UIImage(named: "anim4")
+        loading_5 = UIImage(named: "anim5")
+        loading_6 = UIImage(named: "anim6")
+        
+        images = [loading_1, loading_2, loading_3,loading_4,loading_5,loading_6]
+        
+        animatedImage = UIImage.animatedImage(with: images, duration: 1.0)
+        
+        imageAnim.image = animatedImage
+        
         self.view.backgroundColor = UIColor.white
         
         super.viewDidLoad()
         
         NetworkManager.isUnreachable { networkManagerInstance in
             
-           self.loadingLabel.text = "No internet!"
```

Comment:
You took out this line but left in an empty function. It appears that this feature no longer requires the Reachability functionality, so why are we leaving it in our code base?